package.json

```diff
@@ -11,10 +11,11 @@
   ],
   "description": "Single shot implementation of ridge regression in python for COINSTAC.",
   "dependencies": {
-    "freesurfer-parser": "0.0.2"
+    "freesurfer-parser": "0.0.2",
+    "numeric": "^1.2.6"
   },
   "devDependencies": {
-    "coinstac-simulator": "^2.1.1",
+    "coinstac-simulator": "^2.2.0",
```

Comment:
MRN-Code/coinstac@f2b3d87908305f4422b10f4d61e86a36c5d8369f added some performance improvements. You’ll want to ensure you have the updated dependency by running:

```shell
npm prune && npm install
```